go.mod

```diff
@@ -4,76 +4,84 @@ go 1.23.0
 
 require (
 	github.com/go-logr/logr v1.4.2
-	github.com/onsi/ginkgo/v2 v2.22.0
-	github.com/onsi/gomega v1.36.1
+	github.com/onsi/ginkgo/v2 v2.23.0
+	github.com/onsi/gomega v1.36.2
 	github.com/open-policy-agent/cert-controller v0.12.0
 	github.com/opendatahub-io/opendatahub-operator/v2 v2.10.0
 	github.com/openshift/api v0.0.0-20240904015708-69df64132c91
 	github.com/openshift/client-go v0.0.0-20240904130219-3795e907a202
-	github.com/project-codeflare/appwrapper v1.0.4
+	github.com/project-codeflare/appwrapper v1.1.1
 	github.com/project-codeflare/codeflare-common v0.0.0-20250317102908-1c124db97844
-	github.com/ray-project/kuberay/ray-operator v1.2.2
+	github.com/ray-project/kuberay/ray-operator v1.3.1
 	go.uber.org/zap v1.27.0
-	golang.org/x/exp v0.0.0-20240719175910-8a7402abbf56
-	k8s.io/api v0.31.4
-	k8s.io/apiextensions-apiserver v0.31.2
-	k8s.io/apimachinery v0.31.4
+	golang.org/x/exp v0.0.0-20250305212735-054e65f0b394
+	k8s.io/api v0.32.3
+	k8s.io/apiextensions-apiserver v0.32.3
+	k8s.io/apimachinery v0.32.3
 	k8s.io/client-go v11.0.0+incompatible
-	k8s.io/component-base v0.31.4
+	k8s.io/component-base v0.32.3
 	k8s.io/klog/v2 v2.130.1
-	k8s.io/utils v0.0.0-20240711033017-18e509b52bc8
-	sigs.k8s.io/controller-runtime v0.19.3
-	sigs.k8s.io/kueue v0.10.1
+	k8s.io/utils v0.0.0-20241210054802-24370beab758
+	sigs.k8s.io/controller-runtime v0.20.3
+	sigs.k8s.io/kueue v0.11.0
 	sigs.k8s.io/yaml v1.4.0
 )
 
 replace k8s.io/client-go => k8s.io/client-go v0.31.4
 
+replace k8s.io/api => k8s.io/api v0.31.4
```

Comment:
There are more replace directives here than I'd like, but it was expedient in the short term. Someone who has a better understanding of how to coordinate Kubernetes/OpenShift API versions may be able to do something less heavy handed.